Building prefix dict from the default dictionary ...
Loading model from cache /tmp/jieba.cache
Loading model cost 0.264 seconds.
Prefix dict has been built succesfully.
Saved 10 articles.
Finished Saved 10 articles.
  File "query-classification/word2vec.py", line 43
    output.close()
    ^
IndentationError: expected an indented block
Building prefix dict from the default dictionary ...
Loading model from cache /tmp/jieba.cache
Loading model cost 0.251 seconds.
Prefix dict has been built succesfully.
Building prefix dict from the default dictionary ...
Loading model from cache /tmp/jieba.cache
Loading model cost 0.255 seconds.
Prefix dict has been built succesfully.
Traceback (most recent call last):
  File "query-classification/word2vec.py", line 156, in <module>
    getVec('items')
  File "query-classification/word2vec.py", line 55, in getVec
    senv = np.zeros(300)
NameError: global name 'np' is not defined
Traceback (most recent call last):
  File "query-classification/word2vec.py", line 156, in <module>
    getVec('items')
  File "query-classification/word2vec.py", line 55, in getVec
    senv = np.zeros(300)
NameError: global name 'np' is not defined
Traceback (most recent call last):
  File "query-classification/word2vec.py", line 157, in <module>
    getVec('items')
  File "query-classification/word2vec.py", line 72, in getVec
    senv = np.true_divide(x,cnt)
NameError: global name 'x' is not defined
Traceback (most recent call last):
  File "query-classification/word2vec.py", line 157, in <module>
    getVec('items')
  File "query-classification/word2vec.py", line 72, in getVec
    senv = np.true_divide(x,cnt)
NameError: global name 'x' is not defined
